### PR TITLE
Minh/Ben (readme update)

### PR DESCRIPTION
**Pull request by nhokzminh698**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+- npm install
+
+### Usuage (Both eComs and POS)
+- npm start
+- node server.js
+
+### Link name
+**- POS:** http://localhost:3000/pos
+
+**- eCom:** http://localhost:3000/
```
Comment (gitnamehere) [inline README.md:42]: I don't think we need these at the moment, since we are still deciding on the endpoints

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+- Node.js
+  
+- PostgresSQL
+
+### Initialize
+- npm install
+
+### Usuage (Both eComs and POS)
+- npm start
+- node server.js
```
Comment (gitnamehere) [inline README.md:37]: Will be addressed in a separate PR, but the current scripts are:
Setup Postgres: `npm run setup`
Start server: `npm run start`

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+**- Bonus System:** Loyalty points for frequent customers (discount, bonus items, etc...)
+
+### Extremely Stretchness Goalmericas
+**- Stripe Integration:** The feature allows the use of credit/debit cards transaction on the POS system and website
+
+### Installation
+### Have:
+- Node.js
+  
+- PostgresSQL
```
Comment (gitnamehere) [inline README.md:30]: nitpick: PostgreSQL

**Review by gitnamehere — CHANGES_REQUESTED** [review]

**Review by gitnamehere — APPROVED** [review]
lgtm

**Review by andllrew — APPROVED** [review]

**Pull request merged**